tiatoolbox/cli.py

```diff
@@ -434,5 +440,96 @@ def tissue_mask(
             )
 
 
+@main.command()
+@click.option(
+    "--predefined_model",
+    help="Predefined model used to process the data. the format is "
+    "<model_name>_<dataset_trained_on>. For example, `resnet18-kather100K` "
+    "is a resnet18 model trained on the kather dataset.",
+    default="resnet18-kather100K",
+)
+@click.option(
+    "--pretrained_weight",
+    help="Path to the model weight file. If not supplied, the default "
+    "pretrained weight will be used.",
+    default=None,
+)
+@click.option(
+    "--img_input",
+    help="Path to the input directory containing images to process or an "
+    "individual file.",
+)
+@click.option(
+    "--output_path",
+    help="Output directory where model predictions will be saved.",
+    default="patch_prediction",
+)
+@click.option(
+    "--batch_size",
+    help="Number of images to feed into the model each time.",
+    default=16,
+)
+@click.option(
+    "--return_probabilities",
+    help="Whether to return raw model probabilities.",
```

Comment:
I assume this would save rather than return to console?

Reply:
It is used within the predict method of `CNNPatchPredictor`